### ISSUE TITLE
Contact photo not imported from .vcf file

### ISSUE THREAD

**Issue by tibbi** [SimpleMobileTools/Simple-Contacts#349]

**Comment by jer194** [comment]
First of all: an impressive piece of work, congratulations!

For the bug: contact photo import doesn't seem to work at all (I have dozens of them in one vcf file, none of them worked).

**Comment by AxelFrasch** [comment]
Hello,
I run into the same Problem. This command "PHOTO;TYPE=JPEG:file:///Download/kontaktbilder/Axel.jpg" work on contact-app from Google, delivered with my phone (HTC U12 life Android 8) but not in the simple contact app. I recieved a Java-error (something with null-pointer). Sorry, but  I could not read  more of the errormessage because the message that the Import does'nt work ist in front of the details.
The next I tried, was to put a pictiure to the contact in your app. That works. I exported the contact to a vcf-file and got the encoded Picture in the Exportfile. When I tried to Import These Picture data, Nothing happen, no error but also no Picture.
I hope that will help a Little.

**Comment by SAL-e** [comment]
I run into this bug as well. Here is steps to replicate.
1. Export my contacts from Google using Simple Contacts:
```
BEGIN:VCARD
VERSION:3.0
PRODID:ez-vcard 0.10.5
N:;ADT
TEL;TYPE=OTHER:+1 877 238 2727
PHOTO;ENCODING=b;TYPE=jpeg:/9j/4AAQSkZJRgABAQAAAQABAAD/2wBDAAYEBQYFBAYGBQYH
 BwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2w
 ..................
CATEGORIES:Archive,#
END:VCARD
```
2. Delete contact from Google and import the .vcf file. The picture will not be imported.
3. Export the contacts again and here is the resulting .vcf file.
```
BEGIN:VCARD
VERSION:3.0
PRODID:ez-vcard 0.10.5
N:;ADT
TEL;TYPE=OTHER:+1 877 238 2727
CATEGORIES:Archive,#
END:VCARD
```

There is second path to replicate:
1. In Simple Contacts select and edit the 'ADT' contact (with Picture and stored in Google). Click on storage provider and move it into local db.
2. Using the filter option confirm that contact is now local db. (the Picture is still there.)
3. Next select the contact and tap on 'Edit'. As soon the contact opens in Edit dialog the picture will dispersal. And it is gone even if you cancel the Edit dialog.

**Comment by Russe** [comment]
Same here as well.
Import vcf with pictures into Simple Contacts lead to contacts without pictures.
Importing the same vcf into LineageOS Contacts App is fine, pictures will be imported too and then Simple Contacts will show the pictures.

**Comment by tibbi** [comment]
will be fixed in 6.16.1
